Makefile

```diff
@@ -11,7 +11,7 @@ BUILD_TAG := build
 
 BUILD_IMAGE_NAME := golang:1.9.2
 
-CI_COMMIT_TAG ?= unknown
+CI_COMMIT_TAG ?= $(shell git rev-parse HEAD)
```

Comment:
I might argue to leave this as it was - it appears to be unrelated and at the moment this being unknown gives us a means of determining the origin of artifacts.

Reply:
The reasoning behind this was that we want to ensure again if the version of vault-helper has changed which hasn't been implemented yet on the tarmak side. This was in an effort to differentiate dev versions for testing though with the way vault-helper is downloaded on the tarmak side this might not make much sense. I agree, I guess this is 'ok'